clustersConfig.py

```diff
@@ -21,6 +21,14 @@ def random_mac() -> str:
     return "52:54:" + ":".join(re.findall("..", secrets.token_hex()[:8]))
 
 
+@dataclass
+class ExtraConfigDataType:
```

Comment:
Can we rename it to `ExtraConfigArgs` I think that makes more sense.

Reply:
Yes, updated, this is good to merge now